source/include/FreeRTOS_DNS_Cache.h

```diff
@@ -28,14 +28,12 @@
 #ifndef FREERTOS_DNS_CACHE_H
 #define FREERTOS_DNS_CACHE_H
 
-/* FreeRTOS includes. */
-#include "FreeRTOS.h"
+/* Global Includes & Definitions. */
+#include "FreeRTOS_IP_Common.h"
```

Comment:
adding FreeRTOS_IP_Common to bring in FreeRTOS.h and stdint.h also adds dependencies upon many other modules.  This is a bad idea and encourages spaghetti code.